src/routes/userRoutes.js

```diff
@@ -0,0 +1,69 @@
+const { Router } = require("express");
+const express = require("express");
+const { body, validationResult } = require("express-validator");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+const router = Router();
+
+router.use(express.json());
+//new user registration
+router.post(
+  "/user_create",
+  body("firstName")
```

Comment:
you should create a different middleware for validation into a different file. 
because it gets unreadable.

Reply:
I made a validation with the use of mongoose.